docs/panel/js/pages/games/games.js

```diff
@@ -88,7 +88,9 @@ $(function () {
                     // Add the the box for min bet.
                     .append(helpers.getInputGroup('min-bet', 'number', 'Adventure Minimum Bet', '', e.minBet, 'The minimum amount of points a user can join an adventure with.'))
                     // Add the the box for max bet.
-                    .append(helpers.getInputGroup('max-bet', 'number', 'Adventure Maximum Bet', '', e.maxBet, 'The maximum amount of points a user can join an adventure with.')),
+                    .append(helpers.getInputGroup('max-bet', 'number', 'Adventure Maximum Bet', '', e.maxBet, 'The maximum amount of points a user can join an adventure with.'))
+                    // Add the box for set up the permission who can start the adventure.
+                    .append(helpers.getDropdownGroup('start-permission', 'Permission to start adventure', helpers.getGroupNameById(e.startPermission ?? 7), helpers.getPermGroupNames())),
```

Comment:
Never edit the remote panel. It is done via automation

Reply:
Reverted the change